disputer/index.js

```diff
@@ -30,15 +30,13 @@ async function run(address, pollingDelay, priceFeedConfig, disputerConfig) {
   try {
     // If pollingDelay === 0 then the bot is running in serverless mode and should send a `debug` level log.
     // Else, if running in loop mode (pollingDelay != 0), then it should send a `info` level log.
-    const logObject = {
+    Logger[pollingDelay === 0 ? "debug" : "info"]({
```

Comment:
After figuring out how to apply the override this method was cleaner to apply here based on the serverless logging mode.